test/graphql/types/organizationConnectionList.spec.ts

```
@@ -0,0 +1,151 @@
+import { describe, expect, it, vi, beforeEach } from 'vitest';
+import { builder } from '~/src/graphql/builder';
+import { execute, GraphQLSchema, parse } from 'graphql';
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)